recipes/trodi/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "trodi" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "24d2c526814f6a5d02b37fe1d74efc8eac11965f6973d04267ae78d02fee5d36"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - trodi = trodi.cli:average_and_label
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+    - "numpy =>1.16.*"
+    - "xarray ==0.16.*"
+    - "netCDF4 ==1.5.*"
```

Comment:
Sorry, what I meant was that this set then should just be then:
```yaml
- numpy >=1.16.0
- xarray >=0.16.1
- netcdf4 >=1.5.4
```

Reply:
ok gotcha. Made that change.